pep-0594.rst

```diff
@@ -825,16 +515,6 @@ helpers to pipe the input of one command into the output of another command.
 The module is built on top of ``os.popen``. Users are encouraged to use
 the subprocess module instead.
 
-Module type
-  pure Python
-Deprecated in
-  3.8
-To be removed in
-  3.10
-Has a designated expert
-   no
-Substitute
-  subprocess module
 
 Removed modules
```

Comment:
You could either consider deleting this section, and `formatter` + `fpectl` entirely since they were already removed from the released version of Python, or move them the latter two here—either way, best to be consistent.

Reply:
Resolved in #2274